### DISCUSSION TITLE
Teachers Classroom Creation requires Owner role?

### DISCUSSION THREAD

**Discussion by mbhs-matt** [community/community#141861] [GitHub Education] [answered]
### Hi everyone,

I have Github Education for my organisation (High School) and I'm trying to wrap my head around the permissions. Is it correct that a teacher must be given OWNERSHIP of an organisation to simply [create a classroom](https://classroom.github.com/classrooms/new)? Are there any more granular permisisons that can be given to staff? Giving out owner roles to 150 staff is asking for trouble. It's way more access than they need.

**Comment by muhammadashraf23** [comment, marked as answer]
To create a classroom using GitHub Classroom, a teacher does indeed need to have owner permissions for the organization. Unfortunately, there aren’t more granular permissions within GitHub that allow creating classrooms without granting ownership.

However, there are ways to manage this securely:

Limit Ownership Access Temporarily:

You can assign owner permissions temporarily to teachers when they need to create or manage a classroom, then revert their roles afterward.
Create a Separate Organization for Classrooms:

You could create a separate GitHub organization dedicated to classrooms and educational purposes. This way, you can grant teacher ownership in the educational organization without risking access to the main organization’s sensitive repositories.
Set Up Teams and Roles within the Organization:

While classroom creation requires ownership, you can still use GitHub Teams to control repository access levels for teachers and students more precisely. For instance, teachers could be given maintain or admin roles on specific repositories without being full organization owners.
Use Automation Tools:

You might consider using GitHub Actions or other automation tools to grant and revoke owner permissions as needed.
While the current system isn’t ideal for large-scale deployments with many staff members, these strategies can help minimize risks associated with granting owner roles.

**Reply by mbhs-matt** [reply]
Thanks for the detailed reply! Very helpful thanks!
